README.md

```diff
@@ -92,33 +92,35 @@ Change directory into the freshly cloned repo.
 Once you have cloned and changed directories into the freshly cloned repository
 you have three methods to choose from:
 
-#### Method 1: Using Docker (Safest)*
+#### Method 1: Using Podman (Safest)*
 
-[![Docker Image Version (tag latest semver)](https://img.shields.io/docker/v/qasimwarraich/cli-tutor/latest?label=docker)](https://hub.docker.com/r/qasimwarraich/cli-tutor)
+[![Github Container Registry Image Version (tag latest semver)](https://img.shields.io/docker/v/qasimwarraich/cli-tutor/latest?label=ghcr)](https://ghcr.io/qasimwarraich/cli-tutor:latest)
 
 Prerequisites:
 
 - go v1.18 or higher
-- docker
+- [Podman](https://podman.io/)
 - make (optional)
 
 Instructions:
 
-Option 1: Pull down the image from [Docker Hub](https://hub.docker.com/r/qasimwarraich/cli-tutor)
+Option 1: Pull down the image from [Github Container Registry](https://ghcr.io/qasimwarraich/cli-tutor:latest)
 
-`docker pull qasimwarraich/cli-tutor`
+`podman pull qasimwarraich/cli-tutor`
```

Comment:
This wont work out of the box, the registry name is required.

Reply:
Isn't this already addressed in the other commit? The source was changed to ghcr.

Reply:
The link for the pull command needs to have the full link for the registry, just try it locally and you will see what I am talking about.

Reply:
I'll try it when I'm on the pc. I tried the commands before pushing them. I probably already had the registry locally pulled.

Reply:
Saw my mistake. New commit should fix it. As an aside, the remaining `docker` commands in the `README` file are addressed in the other PR. Didn't want to go fixing them here also.

Reply:
> Saw my mistake. New commit should fix it. As an aside, the remaining `docker` commands in the `README` file are addressed in the other PR. Didn't want to go fixing them here also.

Nice! now just fix the typo in your commit message and we are almost done here.